src/main/java/com/exadel/discountwebapp/discount/vo/DiscountBaseVO.java

```diff
@@ -0,0 +1,32 @@
+package com.exadel.discountwebapp.discount.vo;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
don't need AllArgsContructor - it is too big to use

Can be annotated with @Data

Reply:
done